Sources/Logging/Strings/PurchaseStrings.swift

```diff
@@ -76,6 +76,10 @@ enum PurchaseStrings {
     case sk2_transactions_update_received_transaction(StoreTransaction)
     case sk1_purchase_too_slow
     case sk2_purchase_too_slow
+    case payment_queue_wrapper_delegate_call_sk1_enabled
+
+    // swiftlint:disable:next identifier_name
+    case restorepurchases_called_with_allow_sharing_appstore_account_false
```

Comment:
Nitpick and it was like this before but should this be `restore_purchases_called_with_allow_sharing_appstore_account_false`

Reply:
I think it was like this because the method is `restorePurchases`, so this is one word to signal that.